apps/emqx_authz/test/emqx_authz_api_mnesia_SUITE.erl

```diff
@@ -126,65 +82,149 @@ set_special_configs(_App) ->
 %%------------------------------------------------------------------------------
 
 t_api(_) ->
-    {ok, 204, _} = request(post, uri(["authorization", "sources", "built-in-database", "username"]), [?EXAMPLE_USERNAME]),
-    {ok, 200, Request1} = request(get, uri(["authorization", "sources", "built-in-database", "username"]), []),
-    {ok, 200, Request2} = request(get, uri(["authorization", "sources", "built-in-database", "username", "user1"]), []),
+    {ok, 204, _} =
+        request( post
+               , uri(["authorization", "sources", "built-in-database", "username"])
+               , [?USERNAME_RULES_EXAMPLE]),
```

Comment:
Is that possible to use the swagger's example to do this test?
just like authn?
we can plan to do this in another PR.

Reply:
`emqx_authz_api_mnesia.erl` and `emqx_authz_api_mnesia_SUITE.erl` use the same example in `emqx_authz.hrl`. 
They are provided to swagger, too.

Reply:
Great!